.env.production

```
@@ -0,0 +1,95 @@
+# Production Environment - PRMCMS Consolidated Project
+# Project ID: flbwqsocnlvsuqgupbra (PRMCMS Main)
+# Region: us-east-2
+
+# ======================
+# Supabase Configuration
+# ======================
+VITE_SUPABASE_URL=https://flbwqsocnlvsuqgupbra.supabase.co
+VITE_SUPABASE_ANON_KEY=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6ImZsYndxc29jbmx2c3VxZ3VwYnJhIiwicm9sZSI6ImFub24iLCJpYXQiOjE3NTMzNzY2NzcsImV4cCI6MjA2ODk1MjY3N30.nhWOR862I7FsZ7oOxNnN5-Mp2C9gdhp2TXl84YEPwtw
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The VITE_SUPABASE_ANON_KEY key should go before the VITE_SUPABASE_URL key

(UnorderedKey)